framework/src/main/java/org/tron/tool/litefullnode/LiteFullNodeTool.java

```diff
@@ -527,6 +528,8 @@ private List<String> getCheckpointV2List(String sourceDir) {
   }
 
   private void run(Args argv) {
+    logger.info("LiteFullNodeTool is an obsolete tool and is not recommended"
```

Comment:
LiteFullNodeTool is deprecated and it will be removed in the next major release, use Toolkit.jar db lite instead, parameters are fully compatible.